modin/pandas/test/test_groupby.py

```diff
@@ -2837,8 +2966,14 @@ def eval_rolling(md_window, pd_window):
     eval_general(md_window, pd_window, lambda window: window.std())
     eval_general(md_window, pd_window, lambda window: window.min())
     eval_general(md_window, pd_window, lambda window: window.max())
-    eval_general(md_window, pd_window, lambda window: window.corr())
-    eval_general(md_window, pd_window, lambda window: window.cov())
+    # FIXME: identify error
```

Comment:
Do you plan to identify the error in this PR?

Reply:
yep, done